credsweeper/filters/cred_card_number_check.py

```diff
@@ -0,0 +1,39 @@
+from credsweeper.credentials import LineData
+from credsweeper.filters import Filter
+
+
+class CreditCardNumberCheck(Filter):
+    """Check that value is a credit card number."""
+
+    def run(self, line_data: LineData) -> bool:
+        """Run filter checks on received credential candidate data 'line_data'.
+
+        Args:
+            line_data: credential candidate data
+
+        Return:
+            False, if the sequence is not card number. True if it is
+
+        """
+        if line_data.value is None \
+                or 16 != len(line_data.value) \
+                or ("0" == line_data.value[0] and "0" == line_data.value[1]):
+            return True
+        try:
+            s = 0
+            # https://en.wikipedia.org/wiki/Luhn_algorithm
+            for n in range(0, 16):
+                x = int(line_data.value[n])
+                if not 1 & n:
```

Comment:
```suggestion
                if 0 == 1 & n:
```

Reply:
@csh519 
every second digit has to be multiplied to 2.
If the variant is more readable - let apply this.